Documentation of Comments plugin configuration is more specific #8416 (#8515)
handsontable/handsontable: docs/9.0/guides/cell-features/comments.md

});
```
:::

> ## Set width and height for comment

Reviewer: ```suggestion
## Set a comment box's size
```

> You can use constant dimensions for comment box using `style` parameter.

Reviewer: ```suggestion
To set the width and height of a comment box, use the `style` parameter.
```

::: example #example3
```js
const container = document.querySelector('#example3');

const hot = new Handsontable(container, {
  data: [
    ['', 'Tesla', 'Nissan', 'Toyota', 'Honda', 'Mazda', 'Ford'],
    ['2017', 10, 11, 12, 13, 15, 16],
    ['2018', 10, 11, 12, 13, 15, 16],
    ['2019', 10, 11, 12, 13, 15, 16],
  ],
  rowHeaders: true,
  colHeaders: true,
  contextMenu: true,
  comments: true,
  cell: [

>     { row: 1, col: 1, comment: { value: 'Some comment' } },
>     { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }

Reviewer: ```suggestion
    { row: 1, col: 1, comment: { value: 'Some comment' } },
    // add the `style` parameter
    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
```

  ],
  height: 'auto',
  licenseKey: 'non-commercial-and-evaluation'
});
```
:::

> ## Configure display delay

Reviewer: ```suggestion
## Set a delay for displaying comments
```

> It's also possible to set how many time it will take to show a comment. Just use `displayDelay` parameter.

Reviewer: ```suggestion
To display comments after a pre-configured time delay, use the `displayDelay` parameter.
```

::: example #example4
```js
const container = document.querySelector('#example4');

const hot = new Handsontable(container, {
  data: [
    ['', 'Tesla', 'Toyota', 'Honda', 'Ford'],
    ['2018', 10, 11, 12, 13, 15, 16],
    ['2019', 10, 11, 12, 13, 15, 16],
    ['2020', 10, 11, 12, 13, 15, 16],
  ],
  rowHeaders: true,
  colHeaders: true,

>   contextMenu: true,
>   comments: { 
>     displayDelay: 100,

Reviewer: ```suggestion
  contextMenu: true,
  comments: { 
    // on mouseover, wait 2 seconds before the comment box displays
    displayDelay: 2000,
```
